packages/react-core/src/components/Text/TextContent.tsx

```diff
@@ -12,7 +12,7 @@ export interface TextContentProps extends React.HTMLProps<HTMLDivElement> {
 }
 
 export const TextContent: React.FunctionComponent<TextContentProps> = ({
-  children = null,
+  children,
```

Comment:
Made a couple harmless changes here to force the dist build to re-run so that the previous TextContent.text.tsx file would no longer exist when running jest tests (since it was renamed as a part of this).